src/js/PiskelApi.js

```diff
@@ -70,6 +70,18 @@ var PiskelApi = (function (module) {
     // Hide or show the frame column.
     // Arguments: hideFrameColumn
     TOGGLE_FRAME_COLUMN: 'TOGGLE_FRAME_COLUMN',
+
+    // User selected add new frame
+    // Arguments: none
+    ADD_NEW_FRAME_CLICKED: 'ADD_NEW_FRAME_CLICKED',
+
+    // Add a set of new frames to the current piskel
+    // Arguments: uri, frameSizeX, frameSizeY, frameRate
```

Comment:
Does `frameRate` make sense as an argument for this event?  How does it interact with the existing animation frameRate?